app/operators/fetal_mri_3d_brain_recon_operator.py

```diff
@@ -33,28 +33,26 @@ def compute(self, op_input: InputContext, op_output: OutputContext, context: Exe
 
         # Run motion corrected reconstruction
         if not is_local_testing:
-            if torch.cuda.is_available():
-                cnn_mode = "1"
-                logging.info("SVRTK reconstruction using GPU mode ...")
-            if not torch.cuda.is_available():
-                cnn_mode = "-1"
-                logging.info("SVRTK reconstruction using CPU mode ...")
-
-            motion_correction_mode = "-1"  # -1 minor, 1 severe
-            logging.info("SVRTK reconstruction using Minor motion correction mode ...")
+#            if torch.cuda.is_available():
+#                cnn_mode = "1"
+#                logging.info("SVRTK reconstruction using GPU mode ...")
+#            if not torch.cuda.is_available():
+#                cnn_mode = "-1"
+#                logging.info("SVRTK reconstruction using CPU mode ...")
+#
+#            motion_correction_mode = "-1"  # -1 minor, 1 severe
+#            logging.info("SVRTK reconstruction ...")
 
             subprocess.run([
                 "/home/scripts/docker-recon-brain-auto.bash",
```

Comment:
This line refers to the old recon `.bash` script. It needs to be updated to reflect the new shell script at the correct location in the container.

Reply:
updated, thank you